### PR TITLE
GEN-2470 - styles(combobox): added floating label support

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Combobox: added support for floating label

https://github.com/user-attachments/assets/a81ee57f-7fc4-43d1-8ec9-dbb6f8d0209e

## Justify why they are needed

Necessary for new price calculator page design.

Screenshot 2024-08-13 at 15 45 51

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4586** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4586?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/Combobox/Combobox.css.ts [hunk 1]
```diff
@@ -41,7 +30,34 @@ export const wrapperExpanded = style({
...
+      vars: {
+        [inlinePadding]: '0.875rem',
+      },
+    },
+  ],
+  medium: [
+    inputWrapperBase,
+    {
+      vars: {
+        [inlinePadding]: tokens.space.md,
```
Comment (alebedev) [inline apps/store/src/components/Combobox/Combobox.css.ts:48]: If you assign it in `inputWrapperBase`, you can keep `medium` and `large` styles empty

**Review by alebedev — APPROVED** [review]
Good work!

**Review by alebedev — APPROVED** [review]
Nice!

**Review by gustaveen — APPROVED** [review]
Looks good! 👌
Just wondering if the hover background on items in the combobox should be darker when comparing with Figma? https://www.figma.com/design/vylWsbAVqZTuG8fUEIf0zs/Tiers-%26-Addons?node-id=1958-11840&m=dev

**Comment by guilhermespopolin** [comment]
> Looks good! 👌 Just wondering if the hover background on items in the combobox should be darker when comparing with Figma? https://www.figma.com/design/vylWsbAVqZTuG8fUEIf0zs/Tiers-%26-Addons?node-id=1958-11840&m=dev

It's confusing because another color is used [here](https://www.figma.com/design/5kmmDdh6StpXzbEfr7WevV/Hedvig-UI-Kit?node-id=16259-10128&t=kkTnRBJDQj2zLxZj-4). Gonna check with the design team.

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Aug 14, 6:48 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4586).

**Pull request merged**